Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -39,6 +39,8 @@
 using namespace Gen;
 using namespace PowerPC;
 
+alignas(4096) std::array<u8, CODE_SIZE> Jit64::code_area;
```

Comment:
Could we put the constant "4096" in one place?

I ask because the page size on iOS in some circumstances is 16kb (not sure if it matters for Dolphin), but it is something they changed recently.

I also ask because x86 can sometimes use 2mb/4mb page sizes, and I think there's a (very small) chance the slowdown is caused by forcing the use of 4kb pages, and stressing the TLB. Changing the alignment to 4mb would be a good test to rule out that theory.